src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+interface Area {
+    void draw();
+}
```

Comment:
The interface name `Area` suggests that it should be related to calculating the area, but the method `draw()` inside does not align with this concept. According to the checklist, don't put all behavior into a single interface if the methods are conceptually different from each other. Consider separating `draw()` into its own interface, such as `Drawable`, to adhere to the Single Responsibility Principle.